R/DittoDimPlot.R

```diff
@@ -74,8 +74,10 @@
 #' \code{hover.data} argument is used to determine what data to use.
 #' @param hover.data String vector of gene and metadata names, example: \code{c("meta1","gene1","meta2","gene2")} which determines what data to show on hover when \code{do.hover} is set to \code{TRUE}.
 #' @param hover.data.type Character which, when adding gene expression data to hover, sets the data-type slot that will be obtained. See \link[dittoSeq]{gene} for options.  Default is the \code{data.type} for plotting the main \code{var}, which itself defaults to the \code{"normalized"} data.
-#' @param add.trajectories List vectors representing trajectory paths from start-cluster to end-cluster where vector contents are the names of clusters provided in the \code{trajectories.cluster.meta} input.
+#' @param add.trajectory.lineages List of vectors representing trajectory paths from start-cluster to end-cluster where vector contents are the names of clusters provided in the \code{trajectories.cluster.meta} input.
 #' If Slingshot package was used for trajectory analysis, you can use \code{add.trajectories = SlingshotDataSet(SCE_with_slingshot)$lineages}. In future versions, I might build such retrieval in by default for SCEs.
+#' @param add.trajectory.curves List of vectors representing trajectory paths from start to end where vector contents are x, y coordinates.
+#' If Slingshot package was used for trajectory analysis, you can use \code{add.trajectories = SlingshotDataSet(SCE_with_slingshot)$curves}.
```

Comment:
End with: SlingshotDataSet(SCE_with_slingshot)$curves[,c(dim.1, dim.2)].Note: These curves are not calculated per-dimensionality reduction by dittoSeq, so the dimensions provided to this input need to match dimensions used by dittoDimPlot.

Reply:
or something like that

Reply:
Couldn't do it quite like this, as each element of curves is actually a named list, which contains an element `s` that actually contains the coordinates. Think it should be explicit enough with the changes though.